docs/source/spec/core/stream-traits.rst

```diff
@@ -344,10 +253,10 @@ stream in its output:
             }
         }
 
-Multi-event client behavior
-===========================
+Client Behavior
```

Comment:
Can you make this "Client behavior"? And it seems like this should go after "Initial messages" since it refers to that section before it's introduced.